lib/csu/aarch64c/crt1_c.c

```diff
@@ -76,6 +76,7 @@ _start(void *auxv,
 	void (*cleanup)(void),			/* from shared loader */
 	struct Struct_Obj_Entry *obj)		/* from shared loader */
 {
+	__asm__ volatile(".cfi_undefined c30");
```

Comment:
This has changed upstream in a grosser way by rewriting all of _start in assembly.

It would be really nice if upstream LLVM would support some sort of function attribute to indicate a function that doesn't have a parent frame so that we could annotate these functions with that and LLVM would emit the proper .cfi_undefined directive for the relevant architecture.